packages/extension-ui/src/components/Dropdown.tsx

```diff
@@ -62,7 +65,7 @@ export default styled(Dropdown)`
     -webkit-appearance: none;
     -moz-appearance: none;
     appearance: none;
-    background: ${({ isError, isReadOnly, theme }): string => isError ? theme.inputBackground : (isReadOnly ? '#eee' : '#fff')};
+    background: ${({ isError, isReadOnly, theme }): string => isError ? theme.inputBackground : (isReadOnly ? theme.identicoinBackground : theme.inputBorderColor)};
```

Comment:
```suggestion
    background: ${({ theme }): string => theme.inputBackground};
```
We don't change background for wrong values, also I guess `isReadOnly` is never used, so could be removed